### ISSUE TITLE
[IMPROVEMENTS]: Separate chat for commander

### ISSUE THREAD

**Issue by SimplesOne** [typical-developers/oaklands-feedback#575]
### Roblox Username

AnacondaKillerBest1

### What do you think can have quality-of-life improvements?

The commander has the ability to get you banned, a separate chat that is not sent to roblox would fix this.

### How do you think things can be improved?

https://github.com/typical-developers/oaklands-feedback/assets/47108718/b3ab58ba-9867-4716-ba55-a894c9d281f6
I use c1 to respawn my car from anywhere and t1 for my 2 trailers, I used those commands and got kicked out of the game after a couple minutes for spamming, I had to reactivate my account and who knows, if I do this again I might get actually banned, the chat should still be able to be used for the commander but for repetitive things we should get a separate one

**Comment by fawaz-rizky** [comment]
I agree with this. But for now, you can add emote command (/e) before your command (ex: /e t1) to prevent you to get spawn warned from roblox. It wouldn't allow other people to use your command, though.

**Comment by HDMICIDE** [comment]
thats not how it works at all, if you do /e it just hides the chat, other people can still run the command if you dont have circuit that makes it owner only

**Comment by fawaz-rizky** [comment]
> thats not how it works at all, if you do /e it just hides the chat, other people can still run the command if you dont have circuit that makes it owner only

Are you sure? Have you tried it out?

**Comment by fawaz-rizky** [comment]
Okay, I tested it out again with my alt and it doesnt prevent other player from using it. I guess my friend gave me the wrong command to test it

**Comment by HDMICIDE** [comment]
i mean either way there should be a seperate area for inputting commands because xbox players cant use commanders
